python/desc/imsim/skyModel.py

```diff
@@ -0,0 +1,136 @@
+'''
+Classes to represent realistic sky models.
+Note that this extends the default classes located in
+sims_GalSimInterface/python/lsst/sims/GalSimInterface/galSimNoiseAndBackground.py
+'''
+
+from __future__ import absolute_import, division
```

Comment:
I added this line so that the python 3 checks in the travisc-ci builds would pass.